pisek/__main__.py

```diff
@@ -82,7 +82,7 @@ def add_argument_description(parser):
             "--description",
             "-d",
             type=str,
-            help="Created the dataset with a description of DESCRIPTION.",
+            help="create the dataset with a description DESCRIPTION",
```

Comment:
This should be either `a description of DESCRIPTION` or `the description DESCRIPTION`.